<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect_dialog</class>
 <widget class="QDialog" name="connect_dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Подключение к базе данных</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::DontWrapRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>50</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="name_server_label">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя сервера:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="name_server_combox">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="name_user_label">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя пользователя:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="name_user_combox">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="password_label">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="password_ledit">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="remember_pswd_chebox">
       <property name="text">
        <string>Запомнить пароль (не рекомендуется)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="horizontal_line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_hlayout" stretch="0,0">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>300</number>
     </property>
     <item>
      <widget class="QPushButton" name="connect_button">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Соединить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reject_button">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
